contracts/main/voice/src/ibc.rs

```diff
@@ -84,7 +84,7 @@ pub fn ibc_packet_receive(
     let connection_id = CHANNEL_TO_CONNECTION
         .load(deps.storage, msg.packet.dest.channel_id.clone())
         .expect("handshake sets mapping");
-    Ok(IbcReceiveResponse::default()
+    Ok(IbcReceiveResponse::new(b"")
```

Comment:
As per simon's comment here (https://github.com/AbstractSDK/polytone/pull/7#discussion_r1750259838), we should use `:without_ack()` instead here